docs/notebooks/basic-usage.ipynb

```diff
@@ -88,7 +88,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
```

Comment:
yes, there’s a couple of changes:
- removes all the output that has somehow crept into the file (my fault, sloppily handling git)
- defines `DATA_DIR` in the hidden cell, so that data loading is less sensitive to the working directory in which jupyter lab runs

Reply:
The latter is also thinking ahead about issue #178 (going to comment there about it)

Reply:
Okay, perfect, then all good 👍🏻